db/migrate/20180209163253_split_post_text_into_replies.rb

```diff
@@ -0,0 +1,50 @@
+class SplitPostTextIntoReplies < ActiveRecord::Migration[5.1]
+  def up
+    Post.find_each do |post|
+      post.replies.ordered.each { |reply| reply.update_columns(reply_order: reply.reply_order+1) }
+      reply = Reply.create!(
+        post_id: post.id,
+        user_id: post.user_id,
+        character_id: post.character_id,
+        character_alias_id: post.character_alias_id,
+        icon_id: post.icon_id,
+        content: post.content,
+        created_at: post.created_at,
+        updated_at: post.updated_at,
+        skip_post_update: true
+      )
+      reply.update_columns(reply_order: 0, created_at: post.created_at)
+    end
+    Post.auditing_enabled = false
+    change_table :posts do |t|
+      t.remove :character_id, :character_alias_id, :icon_id, :content
+    end
+    Post.auditing_enabled = true
```

Comment:
Does this need auditing disabled?

Reply:
otherwise we spawn an audit for every. single. post. for those parts being deleted and moved to the reply. If you wanna re-enable it for prod that's sensible but I got fed up with it

Reply:
Huh! I wouldn't have expected audits to act here, since it doesn't have a `current_user`. For prod, I think I'd like it enabled, so we can preserve data alright, but I'm not sure.